src/Microsoft.ML.CodeGen/Templates/Console/ModelOutputClass.cs

```diff
@@ -17,24 +17,20 @@ namespace Microsoft.ML.CLI.Templates.Console
     /// <summary>
     /// Class to produce the template output
     /// </summary>
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "15.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
     public partial class ModelOutputClass : ModelOutputClassBase
     {
         /// <summary>
         /// Create the template output
         /// </summary>
         public virtual string TransformText()
         {
-            this.Write(@"//*****************************************************************************************
-//*                                                                                       *
-//* This is an auto-generated file by Microsoft ML.NET CLI (Command-Line Interface) tool. *
-//*                                                                                       *
-//*****************************************************************************************
-
-using System;
-using Microsoft.ML.Data;
-
-namespace  ");
+if(Target == CSharp.GenerateTarget.Cli){ 
+CLI_Annotation();
+ } else if(Target == CSharp.GenerateTarget.ModelBuilder){ 
+MB_Annotation();
```

Comment:
Instead of having a `if CLI ... else if ModelBuilder, ... else if NextCreatedCallerOfCodeGen`, should we have a parameter which takes set the Annotation?

Our default can be something like: "This is an auto-generated file by ML.NET AutoML". 

This will let future callers set their own annotation, and makes it so we're not listing the set of callers to an API.

---

In addition, the C# style in the ML.NET repo would for the current method be:
```C#
if (Target == CSharp.GenerateTarget.Cli)
{ 
  CLI_Annotation();
} 
else if (Target == CSharp.GenerateTarget.ModelBuilder)
{ 
  MB_Annotation();
}
```

I'm uncertain why StyleCode didn't flag this.